release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1199,6 +1199,26 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Previously, users could not select the `ap-north-east-3` region when using the installation program to create the `install-config.yaml` file. The AWS SDK that caused this problem has been updated, and users can now select the `ap-north-east-3` region. link:https://bugzilla.redhat.com/show_bug.cgi?id=1996544[*BZ#1996544*]
+
+* Previously, installing a private (internal) {product-title} cluster to Azure Stack Hub failed becasuse the installation program did not create DNS record for the API VIP. This update removes the invalid check that caused this problem. The installation program now correctly creates DNS records for private cluster. link:https://bugzilla.redhat.com/show_bug.cgi?id=2061549[*BZ#2061549*]
+
+* Previously, uninstalling an IBM Cloud VPC cluster might have caused unexpected results. When a user uninstalled a cluster (cluster 1), the DNS records of another cluster (cluster 2) where unexpectedlly removed when either the name of cluster 1 (example) was a subset of the the name of cluster 2 (myexample) or both clusters shared a base domain. This update corrects this behavior. Only those resources specific to the cluster that is being uninstalled are now removed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2060617[*BZ#2060617*]
+
+* Previously, Azure stack hub did not support any other disk types other than Standard_LRS. This update adds the ability to customize disk types, which causes the clusters that were created to have a default disk type with no manual customizations. This resulted in making the switch from hard coding the disk type to accepting the inputs from the user and validating it against the Stack Hub APIs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2061544[*BZ#2061544*])
+
+* Previously, the destroyer incorrectly reported the ID of the private route5 hosted zone for the cluster when it deleted the DNS records from the hosted zone. This caused incorrect host zone IDs to be reported in the log of the destroyer. This update uses the proper host zone ID in the log. As a result, the log of the destroyer shows the correct hosted zone ID when destroying the DNS record in the base domain's hosted zone. (https://bugzilla.redhat.com/show_bug.cgi?id=1965969[*BZ#1965969*])
+
+* Previously, system proxy settings were not considered when requesting an AWS custom service endpoint. This update configures AWS custom service endpoint validation to consider the system proxy settings along with a HEAD request to the AWS custom service endpoint. As a result, the AWS custom service endpoint can be accessed from the user's machine. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048451[*BZ#2048451*])
```

Comment:
Do we want a consequence? Something like:
_Previously, because system proxy settings were not considered when requesting an AWS custom service endpoint, the install could fail when using a proxy._